### ISSUE TITLE
how to pass this point?

### ISSUE THREAD

**Issue by lzl124631x** [securas/Daemon-vs-Demon#20]
how to pass this point?

![image](https://user-images.githubusercontent.com/5824517/34332312-1fc78fe6-e8e4-11e7-9d62-6f8473494ad0.png)

**Comment by securas** [comment]
I suggest trying to transform into one of the one eye demons.

**Comment by lzl124631x** [comment]
From the video I could see the only way to pass this point is to transform into eye demon, but once I got caught be demon, it teared me apart and game restarted... Any hint? Thanks.

**Comment by lzl124631x** [comment]
BTW, I'm not sure if it's a bug: If I walk to the north of the beacon shown in the screenshot, and press "DOWN", I will fall and game restart.

**Comment by securas** [comment]
Try to kill the one eye demon before transforming into it.

**Comment by lzl124631x** [comment]
Ah... I just found that I can stand on the demon and press "K" to transform into it... There is no hint about this.

**Comment by securas** [comment]
There are a few hints like the hell demon telling the player that he can transform and the dialog when the first demons are killed... It's always a weird balance on how much to coach the player and how much to leave for discovery. I don't think I've managed to grasp it fully yet. But thanks for the comments.

**Comment by lzl124631x** [comment]
I might have rushed through the dialogs and skipped the hint. As a result, I lingered in the map for long to find a way out, with no luck.
I strongly recommend to add a game scene/routine that forces to player to try out the transform (a very key part of the game) and finish a mini task.
I read the control instruction and knew K was for interaction. By "interaction" I could imagine toggling switch, which is used in the game, but I failed to imagine that I could "K" with a corpse.
Great game anyway. Had a lot of fun!
